diff -ru ffmpeg-3.4/libavformat/mux.c ffmpeg-patched/libavformat/mux.c
--- ffmpeg-3.4/libavformat/mux.c	2017-10-15 08:59:38.000000000 -0700
+++ ffmpeg-patched/libavformat/mux.c	2017-10-31 16:06:35.000000000 -0700
@@ -640,10 +640,9 @@
           st->codecpar->codec_type != AVMEDIA_TYPE_SUBTITLE &&
           st->codecpar->codec_type != AVMEDIA_TYPE_DATA &&
           st->cur_dts >= pkt->dts) || st->cur_dts > pkt->dts)) {
-        av_log(s, AV_LOG_ERROR,
+        av_log(s, AV_LOG_WARNING,
                "Application provided invalid, non monotonically increasing dts to muxer in stream %d: %s >= %s\n",
                st->index, av_ts2str(st->cur_dts), av_ts2str(pkt->dts));
-        return AVERROR(EINVAL);
     }
     if (pkt->dts != AV_NOPTS_VALUE && pkt->pts != AV_NOPTS_VALUE && pkt->pts < pkt->dts) {
         av_log(s, AV_LOG_ERROR,
